File: gen_episodes.py
#!/usr/bin/env python3
"""Generate remaining podcast episodes."""

import os

BASE = "/home/z/my-project/formations"

def write_ep(path, title, niveau_name, ton, c):
    md = f""""# Formation {path.split('/')[1].split('-')[1]} — L'Alchimie du Miroir
# {niveau_name}
## \u00c9pisode {path.split('/')[2].replace('episode-', '').split('-')[0]} : {title}

**Dur\u00e9e cible :** 20 minutes | **Ton :** {ton} | **Public :** Tout public

---

## STRUCTURE

| Segment | Timestamp | Dur\u00e9e | Contenu |
|---------|-----------|-------|--------|
| Accroche | 0:00 | 2:00 | {c['hook']} |
| Enseignement | 2:00 | 10:00 | {c['teaching']} |
| Pratique | 12:00 | 4:00 | {c['practice']} |
| Exercice | 16:00 | 2:00 | {c['exercise']} |
| Cl\u00f4ture | 18:00 | 2:00 | {c['closing']} |

---

## SCRIPT COMPLET

### ACCROCHE (0:00 \u2014 2:00)

{c['hook_full']}

[Pause 3 secondes]

[Transition musicale]

---

### ENSEIGNEMENT (2:00 \u2014 12:00)

{c['teaching_full']}

---

### PRATIQUE (12:00 \u2014 16:00)

{c['practice_full']}

---

### EXERCICE GUID\u00c9 (16:00 \u2014 18:00)

{c['exercise_full']}

[Pause 5 secondes]

---

### CL\u00f4TURE (18:00 \u2014 20:00)

{c['closing_full']}

Votre engagement cette semaine : {c['commitment']}.

*\u00ab {c['quote']} \u00bb* \u2014 {c['quote_author']}

**\u00c0 la prochaine.**

[Fin, musique douce, 8 secondes]

---

## NOTES DE PRODUCTION

**Sources :** {c['sources']}
"""
    os.makedirs(os.path.dirname(BASE + "/" + path), exist_ok=True)
    with open(BASE + "/" + path, 'w') as f:
        f.write(md)


f3n3 = [
    {"file": "formation-3-tout-public/niveau-3/episode-03-pensees-negatives-inversion.md", "title": "Les Pens\u00e9es N\u00e9gatives \u2014 Protocole d'Inversion", "niveau": "Niveau 3 \u2014 Ma\u00eetrise", "ton": "Strat\u00e9gique, lib\u00e9rateur",
     "hook": "Vos pens\u00e9es vous mentent. Et vous les croyez.",
     "hook_full": "Vos pens\u00e9es vous mentent. Et vous les croyez.\n\n[Pause]\n\nLa recherche montre que 80% de vos pens\u00e9es sont n\u00e9gatives et 95% sont r\u00e9p\u00e9titives. Ce n'est pas un d\u00e9faut \u2014 c'est un biais de survie. Le probl\u00e8me : vous les croyez.",
     "teaching": "Le protocole IRIS universel", "teaching_full": "**Le protocole IRIS :**\n\n**I \u2014 Identifier** : Attrapez la pens\u00e9e. *\u00ab Je me dis que... \u00bb*\n\n**R \u2014 Retourner** : Appliquez l'inversion.\n\nLes 5 pens\u00e9es les plus courantes :\n1. *\u00ab Je ne suis pas assez bien \u00bb* \u2192 *\u00ab Je suis en apprentissage \u00bb*\n2. *\u00ab Rien ne change jamais \u00bb* \u2192 *\u00ab Le changement est en cours \u00bb*\n3. *\u00ab Les autres sont meilleurs \u00bb* \u2192 *\u00ab Je suis sur mon propre chemin \u00bb*\n4. *\u00ab C'est trop difficile \u00bb* \u2192 *\u00ab Un pas \u00e0 la fois \u00bb*\n5. *\u00ab Je vais \u00e9chouer \u00bb* \u2192 *\u00ab Chaque tentative est une victoire \u00bb*\n\n**I \u2014 Int\u00e9grer** : Transformez en action. Pens\u00e9e : *\u00ab Je suis nul \u00bb* \u2192 Action : Faites une chose que vous savez bien faire.\n\n**S \u2014 Soutenir** : Ancrez avec la respiration et la gratitude.",
     "practice": "L'inversion en 4 \u00e9tapes", "practice_full": "Le matin, notez votre pens\u00e9e dominante. Appliquez IRIS. Le soir, \u00e9valuez. En crise : respiration (3s) + nommer (3s) + inverser (10s) + agir.",
     "exercise": "Inverser une pens\u00e9e", "exercise_full": "Identifiez une pens\u00e9e n\u00e9gative r\u00e9cente. Appliquez les 4 \u00e9tapes.\n\n[Pause 20 secondes]",
     "closing": "L'inversion n'est pas de la pens\u00e9e magique. C'est un muscle.", "closing_full": "L'inversion n'est pas de la pens\u00e9e magique. C'est un muscle qui se d\u00e9veloppe avec la pratique. M\u00eame si vous n'y croyez pas au d\u00e9but \u2014 faites-le m\u00e9caniquement. Le changement viendra.",
     "commitment": "pratiquer l'inversion au moins 2 fois par jour", "quote": "Ce ne sont pas les \u00e9v\u00e9nements qui nous troublent, mais notre fa\u00e7on de les voir", "quote_author": "\u00c9pict\u00e8te", "sources": "Aaron Beck 1979, National Science Foundation, \u00c9pict\u00e8te"
    },
    {"file": "formation-3-tout-public/niveau-3/episode-04-etats-de-grace.md", "title": "Les \u00c9tats de Gr\u00e2ce", "niveau": "Niveau 3 \u2014 Ma\u00eetrise", "ton": "Contemplatif, inspirant",
     "hook": "Il y a des moments o\u00f9 tout s'aligne. Vous vous sentez vivant.",
     "hook_full": "Il y a des moments o\u00f9 tout s'aligne. La fatigue dispara\u00eet. Vous vous sentez connect\u00e9, pr\u00e9sent, vivant.\n\nCes moments existent dans toutes les traditions \u2014 le flow de Csikszentmihalyi, le satori bouddhiste, la gr\u00e2ce chr\u00e9tienne. Ils ne sont pas r\u00e9serv\u00e9s aux mystiques. Ils sont accessibles \u00e0 tous.",
     "teaching": "Les 7 \u00e9tats universels", "teaching_full": "**1. La paix profonde** \u2014 Un calme au milieu du chaos.\n**2. L'inspiration** \u2014 Une id\u00e9e qui surgit.\n**3. La clart\u00e9** \u00e9 \u2014 Tout devient simple et \u00e9vident.\n**4. L'intimit\u00e9** \u00e9 \u2014 Connexion \u00e0 quelque chose de plus grand.\n**5. L'enthousiasme** \u00e9 \u00e9 \u2014 \u00c9nergie joyeuse.\n**6. Le contentement** \u00e9 \u00e9 \u2014 Satisfaction de l'instant.\n**7. Le flow** \u00e9 \u00e9 \u2014 O\u00f9 le temps dispara\u00eet.\n\n**Les 3 lois :**\n1. Les \u00e9tats ne se forcent pas \u2014 ils se pr\u00e9parent\n2. Les \u00e9tats sont \u00e9ph\u00e9m\u00e8res \u00e9 \u2014 les habitudes restent\n3. Les \u00e9preuves sont les semeuses d'\u00e9tats de gr\u00e2ce",
     "practice": "Cartographier ses \u00e9tats", "practice_full": "Chaque soir pendant 14 jours :\n1. *\u00ab Ai-je v\u00e9cu un \u00e9tat de gr\u00e2ce aujourd'hui ? Lequel ? \u00bb*\n2. *\u00ab Dans quelle circonstance ? \u00bb*\n3. *\u00ab Qu'est-ce qui l'a pr\u00e9c\u00e9d\u00e9 ? \u00bb*\nApr\u00e8s 14 jours, vous aurez votre carte personnelle des \u00e9tats de gr\u00e2ce.",
     "exercise": "Le jardin des \u00e9tats", "exercise_full": "Parcourez votre semaine. Cherchez les moments de gr\u00e2ce. M\u00eamez-les dans un jardin int\u00e9rieur. Remerciez.\n\n[Pause 30 secondes]",
     "closing": "Les \u00e9tats de gr\u00e2ce sont toujours l\u00e0. La question est votre disponibilit\u00e9.", "closing_full": "Les \u00e9tats de gr\u00e2ce ne sont pas des exp\u00e9riences extraordinaires. Ce sont des moments de pr\u00e9sence amplifi\u00e9e. La question n'est pas de les cr\u00e9er \u2014 c'est de les **reconna\u00eetre**.",
     "commitment": "tenir un journal des \u00e9tats de gr\u00e2ce pendant 14 jours", "quote": "La beaut\u00e9 sauvera le monde", "quote_author": "Dosto\u00efevski", "sources": "Csikszentmihalyi, traditions contemplatives"
    },
    {"file": "formation-3-tout-public/nuveau-3/episode-05-meditation-avancee.md", "title": "La M\u00e9ditation Avanc\u00e9e", "niveau": "Niveau 3 \u2014 Ma\u00eetrise", "ton": "Calme, instructif",
     "hook": "La m\u00e9ditation a 3 niveaux. La plupart des gens ne d\u00e9passent jamais le premier.",
     "hook_full": "La m\u00e9ditation a 3 niveaux. Le premier est accessible \u00e0 tous. Le troisi\u00e8me est un tr\u00e9sor que peu connaissent.\n\nEntre les deux, il y a un chemin de transformation progressive.",
     "teaching": "Les 3 niveaux et la s\u00e9ance compl\u00e8te", "teaching_full": "**Niveau 1** \u2014 Observation des pens\u00e9es comme des nuages. Le contenu ne compte pas.\n**Niveau 2** \u00e9 \u2014 Observation de l'observateur. *\u00ab Qui est-ce qui observe ? \u00bb*\n**Niveau 3** \u00e9 \u2014 Repos dans le silence entre les pens\u00e9es.\n\n**La s\u00e9ance de 10 minutes :**\n1. Respiration 4-4-6 (1 min)\n2. Scan corporel (2 min)\n3. Observation (4 min)\n4. Repos dans le silence (2 min)\n5. Retour (1 min)",
     "practice": "Les 5 micro-moments", "practice_full": "1. Avant les emails (10 sec) \u2014 *\u00ab Je suis pr\u00e9sent. \u00bb*\n2. Avant une r\u00e9union (30 sec) \u2014 Respiration + intention\n3. Pendant la pause (30 sec) \u00e9\u2014 Silence total\n4. Face au stress (10 sec) \u00e9\u2014 Respiration\n5. Avant une d\u00e9cision (30 sec) \u00e9\u2014 *\u00ab Si c'est un bien, qu'il se r\u00e9alise. \u00bb*",
     "exercise": "Muraqaba guid\u00e9 de 3 minutes", "exercise_full": "Fermez les yeux. 3 respirations. Parcourez votre corps. Observez vos pens\u00e9es. Reposez-vous dans le silence.\n\n[Pause 20 secondes]\n\nOuvrez les yeux doucement.",
     "closing": "10 minutes le matin. 5 micro-moments. C'est tout.", "closing_full": "La m\u00e9ditation avanc\u00e9e n'est pas un exploit spirituel. C'est une pratique simple et r\u00e9guli\u00e8re qui, avec le temps, transforme votre rapport au moment pr\u00e9sent.",
     "commitment": "pratiquer la s\u00e9ance de 10 min + 3 micro-moments par jour", "quote": "Le silence est la premi\u00e8re de toutes les choses", "quote_author": "Pythagore", "sources": "Thich Nhat Hanh, Kabat-Zinn MBSR"
    },
]

for ep in f3n3:
    ep["niveau"] = "Niveau 3 \u2014 Ma\u00eetrise"
    write_ep(ep["file"], ep["title"], ep["niveau"], ep["ton"], ep)

print(f"F3 N3 episodes 3-5 written")

File: gen-episodes.js
// gen-episodes.js
const fs = require('fs');
const path = require('path');

const BASE = '/home/z/my-project/formations';

function w(path, title, niveau, ton, hook, teaching, practice, exercise, closing, quote, author) {
  const content = `# Formation ${path.split('/')[1]} — L'Alchimie du Miroir
# ${niveau}
## ${path.split('/')[2].split('-').slice(0,2).join('- ')} : ${title}

**Durée cible :** 20 minutes | **Ton :** ${ton} | **Public :** Tout public

---

## STRUCTURE

| Segment | Timestamp | Durée | Contenu |
|---------|-----------|-------|--------|
| Accroche | 0:00 | 2:00 | ${hook} |
| Enseignement | 2:00 | 10:00 | ${teaching} |
| Pratique | 12:00 | 4:00 | ${practice} |
| Exercice | 16:00 | 2:00 | ${exercise} |
| Clôture | 18:00 | 2:00 | ${closing} |

---

## SCRIPT COMPLET

### ACCROCHE (0:00 — 2:00)

${hook}

[Pause 3 secondes]

[Transition musicale]

---

### ENSEIGNEMENT (2:00 — 12:00)

${teaching}

---

### PRATIQUE (12:00 — 16:00)

${practice}

---

### EXERCICE GUIDÉ (16:00 — 18:00)

${exercise}

[Pause 5 secondes]

---

### CLÔTURE (18:00 — 20:00)

${closing}

Votre engagement cette semaine : identifiez votre pratique prioritaire et pratiquez-la.

*« ${quote} »* — ${author}

**À la prochaine.**

[Fin, musique douce, 8 secondes]

---

## NOTES DE PRODUCTION

**Sources :** Synthèse
`;
fs.writeFileSync(path, content);
}

const data = {
  "formation-4-pro-tout-public": [
    ["niveau-1", "Initiation Pro", [
        ["01-pourquoi-spiritualite-pro.md", "Pourquoi la Spiritualité au Travail ?", "Le professionnel moderne est épuisé. Les 5 signes du burnout : fatigue matinale, dimanche trop court, perte de sens, relations qui souffrent, déconnexion de soi. Les 5 pratiques fondamentales.", "Les 5 signes et les 5 pratiques."],
        ["02-fondations-pro.md", "Les Fondations — 5 Règles pour le Pro", "Règle 1 : régularité bat intensité. Règle 2 : micro-pratiques > marathons. Règle 3 : adaptation à la charge. Règle 4 : non-jugement. Règle 5 : patience — résultats en 6-8 semaines.", "Les 5 règles et les 5 piliers pro."],
        ["03-fana-reunions.md", "FANA — L'Effacement Entre Deux Réunions", "Les 3 niveaux du FANA pour le professionnel. Express (30 sec), Standard (2 min), Profond (5 min).", "Les 3 niveaux du FANA pro."],
        ["04-tajalli-bureau.md", "Tajalli — Les Moments de Clarté au Bureau", "Les 7 moments de clarté au travail et le verset-porteur.", "Les 7 Tajalli professionnels."],
        ["05-munajat-decisions.md", "Munajat — Les Décisions Difficiles", "Le dialogue intérieur avec votre sagesse pour les décisions professionnelles. Le protocole en 5 étapes.", "Le protocole Munajat pour les décisions pro."],
        ["06-beance-travail.md", "La Béance — Le Silence au Travail", "Les 3 portes pro : matin (2 min), midi (5 min), retour (3 min). 5 minutes de silence recharge.", "Les 3 portes de la Béance pro."],
        ["07-programme-14-jours-pro.md", "Programme 14 Jours Pro", "14 jours pour intégrer la spiritualité au travail. 3 niveaux de charge : normal (15 min), élevé (10 min), surcharge (5 min).", "Les 14 jours structurés."],
    ], 
    ["niveau-2", "Approfondissement Pro", [
        ["01-burnout-pro.md", "Burnout — Reconnaître, Prévenir, Rebondir", "Le burnout = syndrome clinique (OMS) avec 3 dimensions. Les 4 phases et le protocole de prévention.", "Les 4 phases du burnout."],
        ["02-imposteur-pro.md", "Le Syndrome de l'Imposteur au Travail", "70% des professionnels ont ressenti l'imposteur (Clance & Imes, 1978). 5 signes et 5 solutions.", "Les 5 signes et 5 solutions de l'imposteur pro."],
        ["03-leadership-empathie.md", "Le Leadership par l'Empathie", "L'intelligence émotionnelle compte 2x plus que le QI (Goleman, HBR). Les 4 piliers : auto-conscience, auto-gestion, empathie, compétences sociales.", "Les 4 piliers de l'IE au travail."],
        ["04-conflits-pro.md", "Gérer les Conflits Sans Perdre Sa Dignité", "Le protocole RAPIDE : Respirer, Accueillir, Poser des questions, Investiger, Décider, Exécuter.", "Le protocole RAPIDE."],
        ["05-sens-travail.md", "Le Sens au Travail — Au-Delà du Salaire", "75% des employés désengagés (Gallup). Les 3 dimensions : créativité, contribution, croissance.", "Les 3 dimensions du sens pro."],
        ["06-resilience-pro.md", "La Résilience Professionnelle", "Les professionnels résilients voient l'échec comme des données. 4 piliers : acceptation, sens, connexion, action.", "Les 4 piliers de la résilience pro."],
        ["07-lacher-prise-pro.md", "Le Lâcher-Prise Professionnel", "Les 3 niveaux : résultats, statut, processus.", "Les 3 niveaux du lâcher-prise pro."],
        ["08-seance-n2-pro.md", "La Séance Intégrée Pro", "La routine pro complète : 7h routine (7 min), trajet FANA, 10h Béance, 12h Béance midi, 15h R.A.I.N., 17h scan, 21h journal. 3 niveaux de charge.", "La routine complète et ses 3 niveaux."],
    ],
    ["niveau-3", "Maîtrise Pro", [
        ["01-point-faible-pro.md", "Votre Point Faible Pro", "Le schéma récurrent qui sabote votre carrière. Les 7 profils et le diagnostic.", "Les 7 profils du point faible pro."],
        ["02-presence-pro.md", "La Présence Totale au Travail", "Les 3 piliers : corps, tâche, autre. Les 3 secondes sacrées.", "Les 3 piliers de la présence pro."],
        ["03-pensees-negatives-pro.md", "Les Pensées Négatives Pro", "Le protocole IRIS adapté au bureau.", "Le protocole IRIS pro."],
        ["04-etats-flow-pro.md", "Les États de Flow au Travail", "Les conditions du flow professionnel et comment les créer.", "Les conditions du flow pro."],
        ["05-meditation-avancee-pro.md", "La Méditation Avancée pour le Pro", "Séance du matin + 5 micro-moments pro.", "La séance complète et les micro-moments."],
        ["06-routine-matinale-pro.md", "La Routine du Matin Intégrale du Pro", "La routine complète en 10 minutes et ses 3 niveaux.", "La routine complète du matin pro."],
        ["07-seance-integree-pro.md", "La Séance Intégrée Complète", "3 versions : complète (25 min), express (12 min), urgence (4 min).", "Les 3 versions de la séance intégrée pro."],
        ["08-programme-40-jours-pro.md", "Programme 40 Jours Pro", "40 jours en 4 phases avec 5 piliers quotidiens. Les 10 signes de réussite.", "Les 4 phases et les 5 piliers."],
    ],
};

let count = 0;
for (form, niveaux) in data) {
    for (slug, title, teach, practice) in niveaux) {
        for path in [f"{form}/{niveaux}/{slug}"]:
            count++;
    }
}

console.log(`Created ${count} episodes successfully!`);